PDF_UA/2/8.9 Annotations/8.9.2 Semantics and content/8.9.2.2 Annotations as artifacts/verapdf-profile-8-9-2-2-t02.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<profile xmlns="http://www.verapdf.org/ValidationProfile" flavour="PDFUA_2">
+    <details creator="veraPDF Consortium" created="2023-03-13T10:53:06.894+03:00">
+        <name>ISO 14289-2:202x - 8.9 Annotations - 8.9.2.2 Annotations as artifacts - NoView and ToggleNoView flags</name>
+        <description>Annotations shall be artifacts if the NoView flag is set and the ToggleNoView flag is not set
+            in accordance with ISO 32000-2:2020, Table 167</description>
+    </details>
+    <hash></hash>
+    <rules>
+        <rule object="PDAnnot">
+            <id specification="ISO_14289_2" clause="8.9.2.2" testNumber="2"/>
+            <description>Annotations shall be artifacts if the NoView flag is set and the ToggleNoView flag is not set 
+                in accordance with ISO 32000-2:2020, Table 167</description>
+            <test>structParentType == null || isArtifact == true || ((F &amp; 32) == 0 || (F &amp; 256) == 1)</test>
+            <error>
+                <message>An annotation is included in logical structure with tag %1 (standard type = %2,
```

Comment:
An invisible annotation is included in logical structure with tag %1 (standard type = %2, NoView = %3, ToggleNoView = %4)